babel/messages/catalog.py

```diff
@@ -93,10 +93,10 @@ def __init__(self, id, string=u'', locations=(), flags=(), auto_comments=(),
                        PO file, if any
         :param context: the message context
         """
-        self.id = id #: The message ID
```

Comment:
I was not able to find anything regarding to that in the net, the comments were introduced in 7ab115ce the first time with no comment or reasoning in the commit message. I've pushed a commit on top that removes them.